### PR TITLE
Prepare for release

### PR DESCRIPTION
**Pull request by JoshLove-msft**

### CODE REVIEW

**Comment by JoshLove-msft** [comment]
@jsquire looks like the Codeowners file is still not working as expected. It seems to be only matching on https://github.com/Azure/azure-sdk-for-net/blob/master/.github/CODEOWNERS#L532

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 5.0.0-beta.3 (Unreleased)
-
+## 5.0.0-beta.3 (2021-05-13)
+### Key Bug Fixes
```
Comment (jsquire) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:4]:
```suggestion

### Key Bug Fixes
```

**Review by jsquire — APPROVED** [review]

**Comment by jsquire** [comment]
> @jsquire looks like the Codeowners file is still not working as expected. It seems to be only matching on https://github.com/Azure/azure-sdk-for-net/blob/master/.github/CODEOWNERS#L532

Hrmm.... yeah.  Looks like GitHub takes the first match only.   Lets see if we can't fully qualify that path.

File: sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 # Release History
 
-## 5.0.0-beta.3 (Unreleased)
+## 5.0.0-beta.3 (2021-05-13)
 
+### Key Bug Fixes
```
Comment (JoshLove-msft) [inline sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:5]: Should update to mention the breaking change that was just merged in https://github.com/Azure/azure-sdk-for-net/pull/21054

Comment (pakrym) [reply sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/CHANGELOG.md:5]: MIght not be a bad idea.

**Pull request merged**